src/lint.ts

```diff
@@ -48,3 +48,8 @@ export function createProgram(options: Options): ts.Program {
   const tsconfigPath = path.join(options.targetRootDir, 'tsconfig.json');
   return Linter.createProgram(tsconfigPath);
 }
+
+module.exports = {
```

Comment:
Why is this needed? I think `lint` and `createProgram` are already exported, so the `require('./lint).lint` in `cli.ts` should work without this, unless I am mistaken.

Reply:
Done -- seems like the root problem was simply omitting the `.lint` in the first place.